### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by deadfishhackers** [thelinuxchoice/youbot#4]
**it doesn't give any views**
Things i did to start it
``tor``
``./youbot.sh (and my url here)``
**and they didn't give me any views, i refreshed the page million times but still**

**Comment by JasonJerry** [comment]
For me tor is not installing and also i face the same issue.
Please help me guys !!!

**Comment by EpicSamar360** [comment]
Yeah it's not working, doesn't seem like its getting updated anytime soon either. But there are many other alternative tools out there!

**Comment by deadfishhackers** [comment]
I have tested many tools by they don't work, mind giving a link to a tool
that works?

**Comment by Sanif007** [comment]
> Yeah it's not working, doesn't seem like its getting updated anytime soon either. But there are many other alternative tools out there!

Can u tell me any other tool similar to it? I tried this but it is not working! I mean tool is working but views are not increasing!!
